tasks/test_module/my_task.py

```diff
@@ -0,0 +1,417 @@
+                        writer.writeheader()
+                        continue
+
+                    if section == 'data':
+                        if line.startswith(';'):
+                            continue
+
+                        values = {}
+                        for dimname, dimvalues, dimcnt in dimensions:
+                            values[dimname] = dimvalues[dimcnt]
+                        i = len(dimensions) - 1
+                        while dimensions[i][2] + 2 > len(dimensions[i][1]):
+                            dimensions[i][2] = 0
+                            i -= 1
+                        dimensions[i][2] += 1
+
+                        for i, d in enumerate(line.split(' ')):
+                            values[headers[len(dimensions) + i]] = d
+
+                        writer.writerow(values)
+
+
+    def output(self):
+        return LocalTarget(os.path.join('tmp', classpath(self), '0001.csv'))
+
+
+class FiveYearPopulationColumns(ColumnsTask):
+
+    def version(self):
+        return 1
+
+    def requires(self):
+        return {
+            'seccion_columns': SeccionColumns(),
+            'tags': SubsectionTags(),
+            'sections': SectionTags(),
+            'units': UnitTags(),
+            'license': LicenseTags(),
+            'source': SourceTags()
+        }
+
+    def columns(self):
+        input_ = self.input()
+        spain = input_['sections']['spain']
+        tags = input_['tags']
+        units = input_['units']
+        license = input_['license']['ine-license']
+        source = input_['source']['ine-source']
+        session = current_session()
+        total_pop = input_['seccion_columns']['total_pop'].get(session)
+        columns = OrderedDict([
+            ('gender', OBSColumn(
+                type='Text',
+                name='Gender',
+                weight=0
+            ))
+        ])
+        for i in xrange(0, 20):
+            start = i * 5
+            end = start + 4
+            _id = 'pop_{start}_{end}'.format(start=start, end=end)
+            columns[_id] = OBSColumn(
+                type='Numeric',
+                name='Population age {start} to {end}'.format(
+                    start=start, end=end),
+                targets={total_pop: DENOMINATOR},
+                description='',
+                aggregate='sum',
+                weight=3,
+                tags=[spain, tags['age_gender'], units['people'], license, source]
+            )
+        columns['pop_100_more'] = OBSColumn(
+            type='Numeric',
+            name='Population age 100 or more'.format(
+                start=start, end=end),
+            targets={total_pop: DENOMINATOR},
+            description='',
+            aggregate='sum',
+            weight=3,
+            tags=[tags['age_gender'], units['people'], spain, license, source]
+        )
+
+        return columns
+
+
+class RawFiveYearPopulation(TempTableTask):
```

Comment:
Should be able to subclass [CSV2TempTableTask](https://github.com/CartoDB/bigmetadata/blob/master/tasks/util.py#L1377) instead.